### DISCUSSION TITLE
Ontime tutorials - looking for collaborators

### DISCUSSION THREAD

**Discussion by cpvalente** [cpvalente/ontime#1066] [Q&A]
We are contacted regularly by users who request links for video tutorials around Ontime.

Unfortunately it is unlikely that this would be made by the Ontime developer team (myself included).
We offer our spare time to develop the application and feel this time is better used to continue improving its features.

Ontime has gained a sizeable user base. I am hoping that this effort could be initiated by the user base with documentation-level collaborators.

I am happy to volunteer my help with resources or clarifications. 

If you are interested, you are welcome to reply here or reach out to us at mail@getontime.no

**Comment by ImpossibleWorks** [comment]
I just might take you up on this.

**Reply by cpvalente** [reply]
This would be an incredible contribution, I will help so that this goes smoothly.
Lets chat!

How do you prefer to keep communication?

**Comment by spiketrain** [comment]
Just emailed you. Interested in getting into this with you guys!
